pos_backend_partner/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+POS Backend Partner
+===================
+
+Choose a POS customer from the backend.
+
+
+- Replace customer selection screen in POS by a backend view.
+- Perf improvement : Full customer list is not downloaded anymore by the POS at startup.
+- Benefit from the backend features: advanced search, easy to extend views, etc.
+
```

Comment:
could you add a screenshot of the partner view in PoS ?

Reply:
screenshot added